### DISCUSSION TITLE
How to use /application/o/end-session/

### DISCUSSION THREAD

**Discussion by skalahonza** [goauthentik/authentik#3030] [Q&A] [answered]
Usecase: I am using oAuth2 provider and I would like to logout the user from my app and the authentik.
But when I try to call /application/o/end-session/ I only get 404 not found
I tried that based on this: https://goauthentik.io/docs/providers/oauth2/

maybe a related question here: https://github.com/goauthentik/authentik/discussions/2757 but with no answer

**Comment by BeryJu** [comment, marked as answer]
I've updated the docs, there was a small mistake there, the URL is `/application/o/<application slug>/end-session/`

Now, this endpoint does not revoke any tokens currently, and there is no OAuth2 Token revoke endpoint, however that is on the roadmap: https://github.com/orgs/goauthentik/projects/1

**Reply by crashrip** [reply]
Is there already a revoke endpoint? Or does the logout flow remove the validity of the tokens?
